### DISCUSSION TITLE
Show all the POIs of a particular category

### DISCUSSION THREAD

**Discussion by FinixFighter** [organicmaps/organicmaps#374] [Ideas]
When searching for a specific category, for example "clothes shop", not all the clothes shop are highlighted but only some, depending on the zoom level. It sometimes happens that also a very near (to my position) shop isn't highlighted till I zoom very much! I think that using and OsmAnd like approach is way better: all POIs of the searched category are highlighted, and in case they are too much, some of them have a smaller marker, but you can still see them on the map. Some POI could start to not being highlighted when zoom level is very far, for example when the distance scale (line on the bottom left of OMaps) is about 300 meters.

In the following screenshots you can see the difference between the clothes shop highlighted in OMaps and those highlighted in OsmAnd with the same zoom level and in the same position of my city.
![151478291_247749503607610_2577504175871454587_n](https://user-images.githubusercontent.com/19877271/108185000-1d4c0680-710c-11eb-9ffc-2198c8e31cb5.jpg)
![151774717_182608799892176_5841471368954698955_n](https://user-images.githubusercontent.com/19877271/108185012-20df8d80-710c-11eb-9dad-a6ad39bcb8bd.jpg)

**Comment by bentolor** [comment]
Rendering excess hits as smaller features might be a good option for high zoom levels. 

Some searches might return hits which are very far away (requiring large zoom out) and still in near proximity to each other _or_ large amounts (making it hard to render them explicitly). I just tested the behaviour of Google Maps and was baffled, that they do exactly the same: Omitting hits. IÄd assumed that the the common approach is to render special indicators for multiple hits as single indicator.

So I'd second @FinixFighter : For close-up maps views at a given zoom-level or lower it would be a big improvement to at render all hits in some form.
